official/vision/beta/projects/yolo/configs/experiments/darknet53.yaml

```diff
@@ -0,0 +1,53 @@
+runtime:
+  distribution_strategy: 'mirrored'
+  mixed_precision_dtype: 'float32'
+  loss_scale: 'dynamic'
+task:
+  model:
+    num_classes: 1001
+    input_size: [224, 224, 3]
+    backbone:
+      type: 'darknet'
+      darknet:
+        model_id: 'darknet53'
+  losses:
+    l2_weight_decay: 0.0005
+    one_hot: True
+  train_data:
+    tfds_name: 'imagenet_a'
```

Comment:
that was a copy paste error, for using the same split. when we tested the models, they were different. we will fix that. 

As for the image classification task and TFDS support, we temporarily supported it so that we could test the model because we did not have access to your version of the tfds record. 

Would you like us to add it? We need to change the parser a little bit as well however, because tfds loads the full image, where as your tfrecord decoder loads a string.

Reply:
fixed/addressed